test/library/packages/UnitTest/PopulateTests_PRIM/TestCount.chpl

```diff
@@ -0,0 +1,9 @@
+use UnitTest;
+proc dummy_test(test) throws {
```

Comment:
I would leave a comment indicating what you are testing.  E.g. "verifies that generic single argument functions aren't counted"

Reply:
I have added the comment stating the use of the tests.